src/service/service-directory-api/tests/FamilyHubs.ServiceDirectory.Api.FunctionalTests/CustomWebApplicationFactory.cs

```diff
@@ -53,6 +54,15 @@ public void SetupTestDatabaseAndSeedData()
         {
             var context = scopedServices.GetRequiredService<ApplicationDbContext>();
 
+            // I wish I could just call Migrate() but our migrations don't work on sqlite?
```

Comment:
FYI on this https://learn.microsoft.com/en-us/ef/core/providers/sqlite/limitations#migrations-limitations

TL;DR SQLite sucks (well, Microsoft think so)

Reply:
It's not just a sqlite problem, we have explicitly set mssql specific column types. Maybe we'd run into some other problems after we sorted that, maybe not....